web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -38,6 +56,16 @@ export default class RunComparison extends ReactJS.Component {
       graphKeys: [],
       tableData: [],
       loading: false,
+      generatingPdf: false,
+      modal: {
```

Comment:
The state items on lines 60 and 65 are initialized as objects but are only referenced by their children keys. We can flatten these objects and reference their children keys to improve performance and readability. 

Currently, the `RunComparison` component creates a new item within the component's state as the children keys are not referenced by their parent key. This creates two sets of the same state item: e.g.
```
pdf: {
  pdfHeader: '',
  confirmLoading: false
},
pdfHeader: ''
```

Reply:
> The state items on lines 60 and 65 are initialized as objects but are only referenced by their children keys. We can flatten these objects and reference their children keys to improve performance and readability.
> 
> Currently, the `RunComparison` component creates a new item within the component's state as the children keys are not referenced by their parent key. This creates two sets of the same state item

I will resolve this issue in the next commit.